srcpkgs/carapace/template

```diff
@@ -12,7 +12,7 @@ maintainer="icp <pangolin@vivaldi.net>"
 license="MIT"
 homepage="https://github.com/rsteube/carapace-bin"
```

Comment:
could maybe add changelog from https://github.com/rsteube/carapace-bin/tree/master/docs/src/release_notes or the generated website

Reply:
I added the generated one but it's blank on it's own. You have to navigate to the version with a click.

Reply:
Something broke cross between 0.29.0 & 0.29.1